docs/tempo/website/_index.md

```diff
@@ -8,6 +8,19 @@ aliases:
 
 <p align="center"><img src="logo_and_name.png" alt="Tempo Logo"></p>
 
-Grafana Tempo is an open source, easy-to-use and high-volume distributed tracing backend. Tempo is cost-efficient, requiring only object storage to operate, and is deeply integrated with Grafana, Mimir, Prometheus, and Loki.
+Grafana Tempo is an open source, easy-to-use and high-volume distributed tracing backend. Tempo is cost-efficient, requiring only object storage to operate, and is deeply integrated with Grafana, Mimir, Prometheus, and Loki. Tempo can be used with any of the open-source tracing protocols, including Jaeger, Zipkin, and OpenTelemetry.
```

Comment:
```suggestion
Grafana Tempo is an open source, easy-to-use, and high-volume distributed tracing backend. Tempo is cost-efficient, requiring only object storage to operate, and is deeply integrated with Grafana, Mimir, Prometheus, and Loki. You can use Tempo with any of the open-source tracing protocols, including Jaeger, Zipkin, and OpenTelemetry.
```

Reply:
> high-volume distributed tracing backend. Tempo is cost-efficient, requiring only object storage to operat

In this case, what's the distinction between calling it `backend` and just calling it a database? Database seems more precise (too precise?)

Reply:
I asked the Tempo team wither backend or database was the more appropriate term. @mdisibio noted a preference for backend because database implies SQL, tables, etc. (which Tempo does not have).